### ISSUE TITLE
Mini-browser not rendering `img` and goes blank on refresh

### ISSUE THREAD

**Issue by cam-woloshyn** [scrimba/community#464]
I'm working through the Responsive Web Design Bootcamp.
While viewing **4.11 Building a card component with flexbox - The markup**, the minibrowser wasn't rendering the HTML correctly.

Two issues:

1. The `img` element isn't rendering, which makes it impossible to understand how it's being styled:

![image](https://user-images.githubusercontent.com/6433858/68300184-48236000-006b-11ea-8d99-b4d94eb36a58.png)

2. When I refresh the minibrowser, it goes blank, and stays blank until I do a full page refresh.

![image](https://user-images.githubusercontent.com/6433858/68300244-5d988a00-006b-11ea-89bd-c09830cafe44.png)

I'm on Chrome Version 77.0.3865.120 (Official Build) (64-bit)

**Comment by perborgen** [comment]
Thanks for reporting this one @cam-woloshyn. We'll look into it.

**Comment by cam-woloshyn** [comment]
Following up on this—see video below.

- There should be a background image, which isn't rendered.
- When I refresh the mini-browser, the screen goes blank.

**There's no workaround for this in Scrimba.**

For now, I'm rewriting everything in VSCode so I can follow along. It's tedious and defeats the purpose of doing courses in Scrimba. 

![image](https://screenshot.click/screencast_2019-11-17_07-29-03.gif)

**Comment by perborgen** [comment]
@cam-woloshyn Do you have multiple Scrimba tabs open when this happens?

**Comment by cam-woloshyn** [comment]
@perborgen — sometimes, yes. I have the course page open, as it has an index of all the videos.

I closed that tab, but still having the same issue.
I also cleared my cache, Scrimba cookies, disabled all Firefox add ons and refreshed...still have the issue.

Here's the console output:

<img width="715" alt="image" src="https://user-images.githubusercontent.com/6433858/69048605-e26c9780-09cb-11ea-99fe-bf58fc22865f.png">


FWIW—I've confirmed that it's fetching the assets...
<img width="239" alt="image" src="https://user-images.githubusercontent.com/6433858/69048789-5a3ac200-09cc-11ea-9f43-78c81c13cf4a.png">
